invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/accordion.overrides

```diff
@@ -1,8 +1,17 @@
-.ui.accordion .active.title .icon:not(.button),
-.ui.accordion .accordion .active.title .icon:not(.button) {
+.ui.accordion:not(.invenio-accordion-field) .active.title .icon:not(.button),
+.ui.accordion:not(.invenio-accordion-field) .accordion .active.title .icon:not(.button) {
   &.tab-menu-accordion {
     transform: none;
   }
+
+  transform: @activeIconTransform;
+}
+
+.ui.accordion:not(.invenio-accordion-field) .title .icon,
```

Comment:
congrats on the first PR! 🎇
I have a proposal for this.
Currently, the caret icon in invenio-accodion-field is rotated by js code. Can we make it work with CSS only and remove the js code responsible for that?
it is better to have js code solving more complex stuff than styling :)

Reply:
Sure, I will work on this

Reply:
I think the js for this is in `react-invenio-forms` ([here](https://github.com/inveniosoftware/react-invenio-forms/blob/master/src/lib/forms/AccordionField.js#L192)), should I make a PR there?

Reply:
yes and yes :) 
sorry for not replying straight away but I don't always follow the notifications from github, due to the amount of them, I am usually faster if you ping me on mattermost or discord :)